packages/web-runtime/themes/owncloud/theme.json

```diff
@@ -1,8 +1,18 @@
 {
   "common": {
+    "general": {
       "name": "ownCloud",
-      "slogan": "ownCloud – A safe home for all your data",
-      "logo": "themes/owncloud/assets/logo.svg"
+      "slogan": "ownCloud – A safe home for all your data"
+    },
+    "logo-dark": {
+      "generic": "themes/owncloud/assets/logo_dark.svg"
```

Comment:
What's the point of having a `generic` key instead of going for `"logo": "URL", "logo-dark": "URL", "logo-light": "URL"`?

Also, can I offer my help in finalizing this PR? :)